servers.tf

```diff
@@ -63,52 +69,111 @@ resource "google_compute_instance_template" "vault" {
   depends_on = [google_project_service.service]
 }
 
+############################
+## Internal Load Balancer ##
+############################
+
+resource "google_compute_health_check" "vault_internal" {
+  count   = local.use_internal_lb ? 1 : 0
+  project = var.project_id
+  name    = "vault-health-internal"
+
+  check_interval_sec  = 15
+  timeout_sec         = 5
+  healthy_threshold   = 2
+  unhealthy_threshold = 2
+
+  http_health_check {
```

Comment:
Fun fact: you can replace this with:

```
  https_health_check {
    port         = var.vault_port
    request_path = "/v1/sys/health?uninitcode=200"
  }
```

This obviates the need for a proxy port and nginx, but only in the internal load balancer case (so close!)

Reply:
The original intent was to keep things consistent, but looking at it again, it doens't really make sense to install and use nginx if we don't have to.